proposals/lip-0041.md

```diff
@@ -250,43 +555,101 @@ The Auth module does not execute any logic during the genesis state finalization
 
 #### Transaction Verification
 
-As part of the verification of a transaction `trs`, the following checks are applied. If the checks fail the transaction is invalid and the whole block is discarded.
+As part of the verification of a transaction `trs`, we do the following:
 
-* Check that the transaction nonce is equal to the account nonce by returning `isValidNonce(trs)`.
+* Check that the transaction nonce is equal to the account nonce or to 0 if the account does not exist. If the checks fail the transaction is invalid and the whole block is discarded.
+Notice that a transaction in the transaction pool with a nonce greater than the account nonce is considered "pending" rather than invalid, since it could become valid in the future.
+* Check the signatures property of the transaction [as explained in LIP 0017](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0017.md#signatures-replaces-signature) using `authAccount(address)`, where `address` is the address corresponding to `trs.senderPublicKey`.
```

Comment:
I think this bullet point is outdated and should be removed.

Reply:
removed